internal/releases/charm.go

```diff
@@ -0,0 +1,75 @@
+package releases
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/tidwall/gjson"
```

Comment:
Nice, I almost suggested dropping this and modelling the response as a "proper" type in Go, but this feels like a simple enough addition.

Reply:
Yep, I did that at the start, but it felt a bit overkill; I don't mind putting it back if we think it's better though :)